src/app/filecoin-plus/allocators/utils/formatDatacap.ts

```diff
@@ -0,0 +1,8 @@
+import prettyBytes from 'pretty-bytes'
+
+export function formatDatacap(bytes?: string) {
+  if (bytes === undefined || bytes === '') return 'N/A'
+  const num = Number(bytes)
+  if (isNaN(num) || !Number.isFinite(num) || num === 0) return 'N/A'
```

Comment:
We should use `Number.isNaN` here, instead of the legacy `isNaN`.

As per [MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN): The `isNaN()` function determines whether a value is `NaN`, **first converting the value to a number if necessary.**

```ts
isNaN("Charly")
// true
Number.isNaN("Charly")
// false
```


```suggestion
  if (Number.isNaN(num) || !Number.isFinite(num) || num === 0) return 'N/A'
```